robot/rover/internal_comms/src/CommandCenter.cpp

```diff
@@ -131,14 +138,16 @@ namespace internal_comms
                 free( (void *) &message);
                 //delete &message;
             }
-            //else {
-                //Serial.write(1);
-                //Serial.write(0);
-            //}
-        //}
+            else {
+                Serial.write(1);
+                Serial.write(0);
+                Serial.write(0x0A);
+            }
+        }
```

Comment:
This is probably gonna break, but it would be good to have an "empty" command, so that we don't timeout the while loop every time there is nothing to send